Gemfile

```diff
@@ -22,7 +22,7 @@ gem 'will_paginate', '>= 3.1.7', '< 4'
 gem 'ancestry', '>= 2.0', '< 4'
 gem 'scoped_search', '>= 4.1.7', '< 5'
 gem 'ldap_fluff', '>= 0.4.7', '< 1.0'
-gem 'apipie-rails', '>= 0.5.2', '< 0.6.0'
+gem 'apipie-rails', '>= 0.5.17', '< 0.6.0'
```

Comment:
Can you please open a PR to update the version in packaging? right now we have 0.5.14 packed.

Reply:
@tbrisker, done: https://github.com/theforeman/foreman-packaging/pull/4590